packages/vue/src/utils/index.js

```diff
@@ -132,3 +132,30 @@ export const debounce = (method, delay) => {
 		method();
 	}, delay);
 };
+
+// to check whether arrays are equal in values
+// irrespective of the order of entries
+export const areArraysEqual = (arr1, arr2) => {
```

Comment:
For these types of utilities you might want to consider using an existing library to avoid extra maintenance.
Of course I understand if you have made the technical decision to have these types of functionalities on the library itself giving more freedom towards customization. However this functionality looks like a common one. And possible others too.

Reply:
@mohdashraf010897 Please remove it because we're not using it.